scripts/build-win.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs')
+
+const dir = './apks'
+
+
+async function copyAPK() {
+  if (!fs.existsSync(dir)){
+	fs.mkdirSync(dir)
```

Comment:
you could actually use https://nodejs.org/api/fs.html#fspromisesmkdirpath-options with `recursive` set to true in order to skip directory presence check above